teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -261,6 +305,43 @@ private void submitExitForValidator(final BLSPublicKey publicKey, final int vali
     }
   }
 
+  private tech.pegasys.teku.spec.datastructures.operations.SignedVoluntaryExit generateSignedExit(
+      final BLSPublicKey publicKey, final int validatorIndex) {
+    final ForkInfo forkInfo = new ForkInfo(fork, genesisRoot);
+    final VoluntaryExit message = new VoluntaryExit(epoch, UInt64.valueOf(validatorIndex));
+    final BLSSignature signature =
+        Optional.ofNullable(validatorsMap.get(publicKey))
+            .orElseThrow()
+            .getSigner()
+            .signVoluntaryExit(message, forkInfo)
+            .join();
+    return new tech.pegasys.teku.spec.datastructures.operations.SignedVoluntaryExit(
+        message, signature);
+  }
+
+  private boolean storeExitForValidator(BLSPublicKey blsPublicKey, Integer validatorIndex) {
+    final tech.pegasys.teku.spec.datastructures.operations.SignedVoluntaryExit exit =
+        generateSignedExit(blsPublicKey, validatorIndex);
+    try {
+      SUB_COMMAND_LOG.display("Writing signed exit for " + blsPublicKey.toAbbreviatedString());
+      Files.writeString(
+          voluntaryExitsFolder.toPath().resolve(blsPublicKey.toAbbreviatedString() + "_exit.json"),
+          prettyExitMessage(exit));
+      return true;
+    } catch (IOException e) {
+      SUB_COMMAND_LOG.error("Failed to store exit for " + blsPublicKey.toAbbreviatedString());
+      return false;
+    }
+  }
+
+  private String prettyExitMessage(
```

Comment:
to be an actual prettyPrint you should use something similar to:

https://github.com/ConsenSys/teku/blob/8c172a2eaff954a73a5f665828bf17bdf04c1b6f/teku/src/main/java/tech/pegasys/teku/cli/subcommand/debug/PrettyPrintCommand.java#L109

essentially calling `gen.useDefaultPrettyPrinter()`

Reply:
fair, i was probably lazy about this a little.